### PR TITLE
Add the readme to remind the node version

### PR DESCRIPTION
**Pull request by xiaoshao**
- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/jhipster-registry/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-registry/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ This is the [JHipster](http://www.jhipster.tech/) registry service, based on [Sp
 
 Full documentation is available on the [JHipster documentation for microservices](http://www.jhipster.tech/microservices-architecture).
 
+##Pre-requirements
+
+ 1. plz make sure that nodejs version is between 6 and 9 (I have test it in node8)
```
Comment (deepu105) [inline README.md]: Please change to `Make sure to that your NodeJS version is the latest LTS version supported by JHipster`

Comment (xiaoshao) [reply README.md]: I think the specified version is better.  We could not work based on the latest node10.

Comment (jdubois) [reply README.md]: You need an LTS version - otherwise Node is far too buggy for any serious use

Comment (deepu105) [reply README.md]: Please make the suggested correction. We only support and recommend latest LTS version. Node 10 is not LTS yet

**Pull request closed without merging**